pkg/inst/worker/worker.R

```diff
@@ -1,5 +1,24 @@
 # Worker class
 
+# Utility function to get current system time in milliseconds
+# this code snippet borrowed from [R.utils](cran.r-project.org/package=R.utils)
+currentTimeMillis <- function() {
```

Comment:
I make this similar with java's `System.currentTimeMillis` which returns current datetime in milliseconds but not the process elapsed time.
We may do a little modification on it to avoid the license issue.